font/src/directwrite.rs

```diff
@@ -13,10 +13,7 @@
 // limitations under the License.
```

Comment:
This file should go back to `font/src/directwrite/mod.rs` for platform consistency.